src/main/osd/osd_elements.c

```diff
@@ -1458,7 +1458,13 @@ static void osdElementRssiDbm(osdElementParms_t *element)
 #ifdef USE_RX_RSNR
 static void osdElementRsnr(osdElementParms_t *element)
 {
-    tfp_sprintf(element->buff, "%c%3d", SYM_RSSI, getRsnr());
+    const int8_t antenna = getActiveAntenna();
+
+    if (antenna) {
```

Comment:
Antenna is 0 indexed e.g. Ant 1 = 0.

I haven't tested, but will this only show for Ant 1?

Reply:
Indeed if ant 1 = 0 antenna will fail to the else clause otherwise include the antenna

Reply:
This may either cause constant blinking of `(2)`, or nothing to be shown and cause confusion to user if the element is working.

Reply:
Awaiting test results of that